dbt_project.yml

```diff
@@ -1,12 +1,13 @@
 
 name: 'microsoft_ads_source'
-version: '1.0.0'
+version: '0.2.0'
 config-version: 2
 
 require-dbt-version: [">=0.18.0", "<0.20.0"]
 
 models:
   microsoft_ads_source:
+    +schema: stg_microsoft_ads
     materialized: table
```

Comment:
does this need to be `+materialized`?

Reply:
doesn't really make a difference since the default materialization is a table anyways, but i've noticed that a lot of the old packages don't have a plus sign, which i think is necessary?

Reply:
Good catch, I will update to `+materialized` to be consistent with our more current packages.

Reply:
I agree and updated to `+materialized` to be consistent.